mxcube3/routes/Queue.py

```diff
@@ -640,5 +645,74 @@ def set_autmount():
 
     resp = jsonify({'automount': automount})
     resp.status_code = 200
-    
+
     return resp
+
+@mxcube.route("/mxcube/api/v0.1/queue/mock/diff_plan/<sid>", methods=["GET"])
+def create_diff_plan(sid):
+    '''Juts for creating a diff plan as if it were created by edna and so on.
+    '''
+    from mock import Mock
+
+
+    acq_parameters = mxcube.beamline.get_default_acquisition_parameters()
```

Comment:
Its not very important since its just mockup code, the file suffix and precision should be handled by _set_dc_params.
Which you are calling below, so these few lines should not be necessary.

Reply:
Lines 659 - 662